packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -2058,6 +2060,20 @@ export const BANCOR_REGISTRY_BY_CHAIN_ID = valueByChainId<string>(
     NULL_ADDRESS,
 );
 
+export const BANCORV3_NETWORK_BY_CHAIN_ID = valueByChainId<string>(
+    {
+        [ChainId.Mainnet]: '0xeef417e1d5cc832e619ae18d2f140de2999dd4fb',
+    },
+    NULL_ADDRESS,
+);
+
+export const BANCORV3_NETWORK_INFO_BY_CHAIN_ID = valueByChainId<string>(
```

Comment:
maybe we can be more specific? 

Based on `MixinBancorV3.sol`, it looks like this is the router address, so maybe `BANCORV3_ROUTER_BY_CHAIN_ID`?